lib/postgrex/protocol.ex

```diff
@@ -964,6 +1021,8 @@ defmodule Postgrex.Protocol do
     do: {:commit, nil}
   defp decode_tag("ROLLBACK"),
     do: {:rollback, nil}
+  defp decode_tag( :suspended ),
+    do: {:select, nil}
```

Comment:
It may not be select and well need to save the raw tag and return the correct rows as we go.